added meta html values (#3213)
plone/volto: docs/source/configuration/expanders.md

---
html_meta:

>   "description": "Learn to configure the API expanders Volto uses using the `settings.apiExpanders` like this"

Reviewer: ```suggestion
  "description": "Configure the API expanders in Volto using the `settings.apiExpanders`"
```

>   "property=og:description": "Learn to configure the API expanders Volto uses using the `settings.apiExpanders` like this"

Reviewer: ```suggestion
  "property=og:description": "Configure the API expanders in Volto using the `settings.apiExpanders`"
```
